app/controllers/users_controller.rb

```diff
@@ -1,21 +1,30 @@
 class UsersController < ApplicationController
-  def new
-    @user = User.new
+  def index
+    @users = User.paginate(page:params[:page], per_page: 10)
   end
 
-  def show
-    @user = User.find(params:[:id])
+  def new
+    @user = User.new
   end
 
   def create
-    @user = User.new(parmas[:user])
+    @user = User.new(user_params)
     if @user.save
-    flash[:success]="Saved successfully!"
-    redirect_to login_url
+      flash[:success]="Saved successfully!"
+      redirect_to static_pages_url
```

Comment:
This is causing an error because route doesn't exist.
To check available routes do command: **docker-compose exec web rails routes**
![Screen Shot 2021-04-27 at 10 08 13 PM](https://user-images.githubusercontent.com/74635797/116255984-573d2700-a7a5-11eb-85fa-08bfe1c66e17.png)